File: problem_forty_one.js
var inp  = 5
function MyFunction(inp){
    for(var i = inp ; i > 0; i--){
        if(i >= 2){
             console.log(2,"^",i)
        }
        else{
            console.log("2+1 ")
        }
    }
}
MyFunction(inp)

File: problem_forty_one.py
inp = int(input("Enter the number:"))
def MyFunction(inp):
    for i in range(inp,0,-1):
        if i >=  2:
           print("{}{}{} ".format(2,"^",i,),end= "+")
        elif i <=2:
           print(" 2 + {}".format(i),end ="")   

MyFunction(inp)